jupyterhub_config.py

```diff
@@ -143,6 +143,7 @@ def finalize_user_content(spawner):
     - adding date_culled to the TrackingItem given by FF_TRACKING_ID
     """
     # grab this info fresh every time
+    err_output = []
```

Comment:
what does this line do? just resets any existing errors that have been reported?

Reply:
Missing declaration, so was a syntax error that made it through.